packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -266,11 +266,7 @@ const renderApp = (args) => {
     // Do *not* add to these without a very good reason - globals are a liability.
     const windowGlobals = {
         __DEVICE_TYPE__: deviceType,
-        __PRELOADED_STATE__: appState,
-        __ERROR__: error,
-        // `window.Progressive` has a long history at Mobify and some
-        // client-side code depends on it. Maintain its name out of tradition.
-        Progressive: getWindowProgressive(req, res)
```

Comment:
what's the reason behind removing `window.Progressive`? I don't believe this is publicly documented, but this is a breaking change...

Reply:
PR has been updated to keep window.Progressive for now.